modelon/impact/client/experiment_definition.py

```diff
@@ -206,6 +216,87 @@ def with_modifiers(self, modifiers=None, **modifiers_kwargs):
             )
         return new
 
+    def with_extensions(self, *experiment_extension):
```

Comment:
I think it will be more flexible if the input is just a list rather than *args, because if we want to add more input arguments it will be very hard as it is now.